spec/spider-gazelle/helper_spec.cr

```diff
@@ -55,6 +64,41 @@ class HelperSpecActionController < ActionController::Base
       text "Ouch.", schema: String.to_openapi_schema
     end
   end
+
+  @[OpenAPI(
+    <<-YAML
+      summary: Destroy route
+    YAML
+  )]
+  def destroy
+    head :no_content, description: "No content available"
+  end
+
+  get "/custom_route", :custom_route do
+    redirect_to("/hello/:id", description: "custom route redirect to show")
+  end
+
+  @[OpenAPI(
+    <<-YAML
+      summary: custom route
+    YAML
+  )]
+  def custom_route
+    previous_def
+  end
```

Comment:
Is this because the routes are blocks?

Reply:
Route annotation works by annotating route methods, however this is a macro, so we can't annotate it

```crystal
get "/custom_route", :custom_route do
  redirect_to("/hello/:id", description: "custom route redirect to show")
end
```

However we can achieve the same result for these custom routes (non-CRUD) by redefining it and annotating over the redefined method
```crystal
   @[OpenAPI(
     <<-YAML
       summary: custom route
     YAML
   )]
   def custom_route
     previous_def
   end
```